paper.md

```diff
@@ -1,53 +1,118 @@
-and partly because data analysis for individual telescopes
-typically used just one of the standards along with the associated analysis
-code. New low frequency instruments (e.g. MWA (http://www.mwatelescope.org/),
-PAPER (http://eor.berkeley.edu/), HERA (http://reionization.org/)),
-have required custom analysis and simulation software that rely on a range of
-different file formats. pyuvdata was designed to facilitate interoperability
-between these instruments and codes by providing high quality, well documented
-conversion routines as well as an interface to interact with interferometric
-data and simulations directly in python.
+converting between them in a stable, repeatable way has historically been
+challenging.  This is partially due to conflicting assumptions and standards, giving
+rise to significant (though sometimes subtle) differences between formats.
+Interfacing with different data formats -- like one does when they convert from one
+format to another -- thus requires careful accounting for the complex mathematical
+relationships between both data and metadata to ensure proper data fidelity. This is
+required both for leveraging existing community-favored tools that typically built
+to interface with a specific data format, as well as analyses requiring bespoke tools
+for specialized types of analyses and simulations leveraging data in a variety of
+formats.
+
+pyuvdata has been designed to facilitate interoperability between different instruments
+and codes by providing high quality, well documented conversion routines as well as an
+interface to interact with interferometric data and simulations directly in Python.
+Originally motivated to support new low frequency instruments (e.g. MWA
+(http://www.mwatelescope.org/), PAPER (http://eor.berkeley.edu/), HERA
+(http://reionization.org/)), the capabilities of pyuvdata have been steadily expanded
+to support handling of data from several telescopes, ranging from meter to submillimeter
+wavelengths, including SMA, ALMA, SMA, VLA, ATCA, CARMA, LWA, among others.
+
+# Major updates in this version
+In the time since it was initially published [@pyuvdata_v1], pyuvdata has undergone a
+significant expansion in capabilities. In addition to general performance improvements
+and restructuring, the newest version of pyuvdata includes several new major features,
+including:
 
-pyuvdata currently supports reading and writing uvfits [@uvfits] and
-miriad [@miriad] files and reading FHD [@fhd] visibility save files.
+- The addition of the `UVCal` class, which provides a container for handling calibration
+solutions (bandpass, delays, and gains) for interferometric data. Supported data formats
+include MS, FHD, CalFITS, and CalH5.
+- The addition of the `UVBeam` class, which provides a container for handling models
+of the primary beam for antennas within an interferometric array. Supported data formats
+include BeamFITS and MWA.
```

Comment:
and CST

Reply:
Added in f0dccaf30c17cf2781aee89e501427e416881d44.